spec/models/html_variant_spec.rb

```diff
@@ -55,4 +55,16 @@
     cloudinary_string = "/c_limit%2Cf_auto%2Cfl_progressive%2Cq_auto%2Cw_420/https://devimages.com/image.jpg"
     expect(html_variant.html).to include(cloudinary_string)
   end
+
+  it "does not add prefix if cloudinary already start" do
```

Comment:
_Non-blocking, clarifying, nitpick_: 
```suggestion
  it "does not add prefix if it already starts with cloudinary" do
```